tests/test_authorizer.py

```diff
@@ -341,6 +374,30 @@ def test_refresh(self):
         self.assertEqual(set(["*"]), authorizer.scopes)
         self.assertTrue(authorizer.is_valid())
 
+    def test_refresh__with_invalid_username_or_password(self):
+        authorizer = prawcore.ScriptAuthorizer(
+            self.authentication, USERNAME, "invalidpassword"
+        )
+        with Betamax(REQUESTOR).use_cassette(
+            "ScriptAuthorizer_refresh__with_invalid_username_or_password"
+        ):
+            self.assertRaises(prawcore.OAuthException, authorizer.refresh)
+            self.assertFalse(authorizer.is_valid())
+
+    def test_refresh_with__invalid_otp(self):
```

Comment:
Since the diff shows this as aded, its name _should_ be `test_refresh__with_invalid_otp` which would mean it should be above the previously added test.

Reply:
Sorry, the diff is a bit of a mess. I added two tests to 111, `test_refresh_with__valid_otp` and `test_refresh_with__invalid_otp`, but the tests weren't sorted lexicographically.

Here is the repository's current order for the tests in the `ScriptAuthorizerTest`

    test_initialize__with_untrusted_authenticator
    test_refresh
    test_refresh_with__valid_otp
    test_refresh_with__invalid_otp
    test_refresh__with_invalid_username_or_password

I added `test_refresh_with_scopes` to that section,  and switched the otp tests. Here is the order in which I put them in this PR:

    test_initialize__with_untrusted_authenticator
    test_refresh
    test_refresh__with_invalid_username_or_password
    test_refresh_with__invalid_otp
    test_refresh_with__valid_otp
    test_refresh_with_scopes

Would you like me rename and reorder them to

    test_initialize__with_untrusted_authenticator
    test_refresh
    test_refresh__with_invalid_otp
    test_refresh__with_invalid_username_or_password
    test_refresh__with_scopes
    test_refresh__with_valid_otp

or something else?

Also, there is one test added to this section in 120, which is named `test_refresh_with_retries`. Should that test be renamed to `test_refresh__with_retries`, or something else?

Reply:
> Would you like me rename and reorder them to...

Yes please. Thanks for cleaning that up.

Reply:
I would like to go through entire repo and reorder all methods. Maybe I can write a tool to do it.